src/main/java/io/curity/identityserver/plugin/aws/config/AWSAuthenticatorPluginConfig.java

```diff
@@ -36,18 +42,22 @@
     @Description("Secret key used for communication with aws")
     String getClientSecret();
 
-    @Description("URL to the AWS authorization endpoint")
-    @DefaultURI("/oauth2/authorize")
-    URI getAuthorizationEndpoint();
-
-    @Description("URL to the AWS token endpoint")
-    @DefaultURI("/oauth2/token")
-    URI getTokenEndpoint();
-
-    @Description("A space-separated list of scopes to request from AWS")
+    @Description("Scopes must be separated by spaces. System reserved scopes are openid, email, phone, profile, and aws.cognito.signin.user.admin.")
```

Comment:
why use this approach rather than boolean flags? Does Cognito support an open-ended scope configuration and so this allows an admin to configure any matching scope? Also, are all of those "reserved"? I only saw special handling for `openid` and `profile`.

Reply:
let's discuss it, and I will create another PR for it.